helm/ffc-demo-web/jenkins-aws.yaml

```diff
@@ -0,0 +1,14 @@
+environment: production
+name: aws-ffc-demo
+imagePullSecret: myregistrykey
+service:
+  port: 443
+  type: NodePort
+container:
+  allowPrivilegeEscalation: true
+  imagePullPolicy: Always
+ingress:
+  alb:
+    tags: Name=FFCLDNALBFOC001,Environment=FFC,Project=FFC,Region=eu-west-2,Location=London,ServiceCode=FFC,ServiceType=LOB
+    arn: arn:aws:acm:eu-west-2:562955126301:certificate/d9bd0a19-487e-4bd1-beb8-a26806739c9a
+    securityGroups: sg-06d6c4e13e3de5a5e
```

Comment:
I think these details should be moved into pipeline parameters or environment variables so that the details of infrastructure aren't embedded in the source for the application.

Reply:
these have been removed now